configs/14.0/deb/monolithic/devel-dbg.postrm

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright 2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+case "${1}" in
+	remove|purge)
+		find -L __AT_DEST__ -path '*/.debug/*' -type l -exec rm -f {} \;
```

Comment:
This is removing just broken symlinks, which is what we expect to remove in case someone removes/purges just one debuginfo package.
OK.